### PR TITLE
[MIG] mgmtsystem_action: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- `mgmtsystem`: https://github.com/OCA/management-system/pull/365

### CODE REVIEW

File: mgmtsystem_action/views/mgmtsystem_action.xml [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+                                        </div>
+                                        <div class="oe_kanban_bottom_right">
+                                            <img
+                                            t-att-src="kanban_image('res.users', 'image_small', record.user_id.raw_value)"
+                                            t-att-title="record.user_id.value"
+                                            t-att-alt="record.user_id.value"
+                                            width="24"
+                                            height="24"
+                                            class="oe_kanban_avatar"
+                                        />
```
Comment (michaelslade54) [inline mgmtsystem_action/views/mgmtsystem_action.xml]:
```suggestion
                                            <field name="user_id" widget="many2one_avatar_user"/>
```

**Comment by YvanDotet** [comment]
Hello
Is it ready to merge in the 14.0 branch ?
Thank you.

**Review by simahawk — APPROVED** [review]

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+odoo14-addon-mgmtsystem @ git+https://github.com/dzungtran89/management-system@14.0-mig-mgmtsystem#subdirectory=setup/mgmtsystem
```
Comment (simahawk) [inline test-requirements.txt:1]: please drop this

**Comment by simahawk** [comment]
to speed it up as this a dependency for other PRs: https://github.com/OCA/management-system/pull/382

**Pull request closed without merging**